yarn-error.log

```diff
@@ -0,0 +1,346 @@
+Arguments: 
+  /usr/local/Cellar/node/13.2.0/bin/node /usr/local/bin/yarn build
```

Comment:
Please add `yarn-error.log` to the git ignrores.

Reply:
Done